package.json

```diff
@@ -76,5 +83,6 @@
     "extends": [
       "@commitlint/config-conventional"
     ]
-  }
+  },
+  "packageManager": "yarn@4.2.2"
```

Comment:
Are we really using `yarn 4` for this repo? Is there no side effect for this?

Reply:
Works fine with yarn v4, at least in the "worker mode". As far I understand, [corepack](https://yarnpkg.com/corepack) (binary installed with the NodeJS) makes sure that yarn version from the `packageManager` field is used within the project.

Reply:
As long as it is supported within Cloudflare then we are good to go I guess!

Reply:
So far I had an okay experience with `bun` except for testing. `yarn@4` is fine to me as long as we don't use `Zero Install` which implies committing the whole node modules list. The main concern with yarn > 1 (which also applies to bun actually) is compatibilty with tools. For example, Vercel would accept neither (only yarn classic, npm, pnpm), so would AWS etc. Github actions don't seem to mind, neither does Cloudflare.